flink-python/pyflink/table/table_result.py

```diff
@@ -140,9 +141,80 @@ def get_table_schema(self) -> TableSchema:
         :rtype: pyflink.table.TableSchema
 
         .. versionadded:: 1.11.0
+        .. deprecated:: 2.1.0
+           Use :func:`TableResult.get_resolved_schema` instead.
         """
         return TableSchema(j_table_schema=self._get_java_table_schema())
 
+    def get_resolved_schema(self) -> ResolvedSchema:
+        """
+        Get the resolved schema of result.
+
+        The schema of DDL, USE, EXPLAIN:
```

Comment:
Where does this part come from? It seems not in the Java code. Drop this?

Reply:
Nvm it seems I checked the wrong class. Not sure if this doc is really useful. It will always be incomplete and not covering all cases. I think we can remove it also in Java.